M2/Macaulay2/m2/exports.m2

```diff
@@ -328,6 +330,9 @@ export {
 	"Range",
 	"RealField",
 	"Reduce",
+	"References",
+	"RegexPOSIX",
+	"RegexPerl",
```

Comment:
These two won't be needed.

Reply:
This is added as a part of the regex PR. They show up in the diff because this PR is based on that one. I'll discuss it there.